src/Agent/MyStatistics.vue

```diff
@@ -1,71 +1,129 @@
-    </b-row>
-    <b-row>
-      <b-col>
-        <b-table style="margin-top:10px" small bordered
-          :items="statistics"
-          :fields="fields">
-          <template slot="statesCounts" slot-scope="data">
-            <div style="background-color: #dbeffa">
-              <b-progress-bar variant="primary" :value=data.item.statesCounts.release :max="data.item.agents" show-progress><div class="agent-state-text" style="min-width:100%; color:black">Released</div></b-progress-bar>
-            </div>
-            <div style="margin-top:2px; background-color: #ffeeba">
-              <b-progress-bar variant="warning" :value="data.item.statesCounts.available" :max="data.item.agents" show-progress><div class="agent-state-text" style="min-width:100%; color:black">Available</div></b-progress-bar>
-            </div>
-            <div style="margin-top:2px; background-color: #ffeeba">
-              <b-progress-bar variant="warning" :value="data.item.statesCounts.ringing" :max="data.item.agents" show-progress><div class="agent-state-text" style="min-width:100%; color:black">Ringing</div></b-progress-bar>
-            </div>
-            <div style="margin-top:2px; background-color: #e2fada">
-              <b-progress-bar variant="success" :value="data.item.statesCounts.oncall" :max="data.item.agents" show-progress><div class="agent-state-text" style="min-width:100%; color:black">Oncall</div></b-progress-bar>
-            </div>
-            <div style="margin-top:2px; background-color: #ffeeba">
-              <b-progress-bar variant="warning" :value="data.item.statesCounts.wrapup" :max="data.item.agents" show-progress><div class="agent-state-text" style="min-width:100%; color:black">Wrapup</div></b-progress-bar>
-            </div>
-          </template>
-          <template slot="occup" slot-scope="data">
-            <div style="text-align:center;">{{data.item.occup.oncall}}</div>
-            <!--<div class="session-state-text"><b>oncall: </b>{{data.item.occup.oncall}}</div>
-            <div class="session-state-text"><b>ringing: </b>{{data.item.occup.ringing}}</div>
-            <div class="session-state-text"><b>available: </b>{{data.item.occup.available}}</div>
-            <div class="session-state-text"><b>release: </b>{{data.item.occup.release}}</div>-->
-          </template>
-        </b-table>
-      </b-col>
-    </b-row>
-  </b-collapse>
-</div>
+  <div v-access:myStatistics-widget>
+    <toggle-bar style="cursor: move"></toggle-bar>
+    <b-collapse v-model="showCollapse" id="collapseMyStatistics" class="mt-2">
+      <b-container style="margin-top:10px">
+        <b-row>
+          <b-col sm="4">
+            <b-form-select class="pointer" size="sm" v-model="period.value" @change="set_period">
+              <option v-for="period in periods" :value="period.value">{{period.name}}</option>
+            </b-form-select>
+          </b-col>
+          <b-col style="min-width: 300px" sm="8">
+            <multiselect
```

Comment:
We kind of have a common skill selector (as a tag line), why to have another one?